src/components/Dashboard/Alert/fixtures/AlertsBody.fixture.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 
 import { AlertsBody } from '../AlertsBody';
-import { criticalAlert, unknownTypeAlert, warningAlert } from './AlertItem.fixture';
+import { cephDiskInaccessibleAlert, watchDogAlert, cephDataRecoveryAlert } from './AlertItem.fixture';
 import { AlertItem } from '../AlertItem';
 
 export default {
   component: AlertsBody,
   props: {
     children: [
-      <AlertItem key="1" alert={criticalAlert} />,
```

Comment:
why changing these generic alerts to specific ones ?

Reply:
Sorry, I missed it. I thought I am changing in storage tab. Thanks

made the required changes.